### ISSUE TITLE
local search method sw used in autodock gpu is similar to psw for autodock cpu?

### ISSUE THREAD

**Issue by srilekha1993** [ccsb-scripps/AutoDock-GPU#264]
As our aim is to compare the energy values of autodock cpu and gpu but autodock gpu is not supported by psw algorithm and autodock cpu is not sw algorithm. So if we use psw algorithm in autodock cpu and sw algorithm in autodock gpu will it be a correct comparison?

**Comment by diogomart** [comment]
No, the Solis-Wets (SW) in AutoDock-GPU is far superior to SW of AutoDock4. I think the difference is the magnitude of the changes in the coefficients that define the conformation (referred to as genes in genetic algorithms).

**Comment by diogomart** [comment]
The energies should be very similar for the same pose.
